def.cf

```diff
@@ -258,6 +258,11 @@ bundle common def
         comment => "If reports are not collected for an extended period of time
                     the disk may fill up or cause additional collection
                     issues.";
+
+      "enable_inventory_ipv4_addresses" -> { "Mission Portal" }
```

Comment:
Yes, it should be performant, and for Enterprise users it makes sense to have it on by default, but it arguably provides very little benefit for non enterprise users.

Classes being in def.cf are less important to me than variables, the classes defined in def.json would apply regardless of where we document them in policy. In which file do you suggest this class be set?